### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by Time6628** [LapisBlue/Pore#34]
Getting this error when the plugin is enabled:
https://gist.github.com/Time6628/061e9077976385011ccb

**Comment by unascribed** [comment]
Ensure you are using the **latest** build of Sponge and not the recommended build.

**Comment by Time6628** [comment]
I am using the latest build of forge, sponge, and pore.

**Comment by jamierocks** [comment]
Could you post your full log?
I would like to know exactly when this exception was thrown.

**Comment by Time6628** [comment]
https://gist.github.com/Time6628/f0266e9fc410bb59e12bd28db1026723

**Comment by jamierocks** [comment]
Right, so I'm pretty sure I know why the issue is occuring, but I need the entire log to be sure, at least up to the stack trace.

**Comment by jamierocks** [comment]
Actually I should have enough to prove whether my theory is correct.

**Comment by jamierocks** [comment]
It looks as if Sponge isn't firing the GamePreInitializationEvent for Pore. I'll take a proper look in the morning, however I should note that Pore doesn't really play nicely with mods anyway.
